make/autoconf/toolchain.m4

```diff
@@ -35,19 +35,18 @@
 m4_include([toolchain_microsoft.m4])
 
 # All valid toolchains, regardless of platform (used by help.m4)
-VALID_TOOLCHAINS_all="gcc clang xlc microsoft"
+VALID_TOOLCHAINS_all="gcc clang microsoft"
```

Comment:
This file should not be changed: We still use xlc on AIX.

Reply:
Restored